packages/scm/src/browser/scm-contribution.ts

```diff
@@ -179,8 +186,33 @@ export class ScmContribution extends AbstractViewContribution<ScmWidget> impleme
             });
             registry.registerItem(item);
         };
+
+        const registerCollapseAllItem = (command: Command) => {
```

Comment:
what is the point of the function and arg, if it is called only once with the same argument?